src/main/java/org/sonarsource/sonarlint/ls/backend/BackendServiceFacade.java

```diff
@@ -62,6 +62,8 @@
 
 public class BackendServiceFacade {
 
+  public static final String MONITORING_DISABLED_PROPERTY_KEY = "sonarlint.monitoring.disabled";
```

Comment:
This property is set in the SLLS tests and will also be set in the SLVSCODE tests 🙈

Reply:
Let's reverse the name, it will make everything much easier to follow. i.e. `MONITORING_ENABLED_PROPERTY_KEY = "sonarlint.monitoring.enabled"`

Reply:
I'm half-and-half on this one.

I agree that it would make it somewhat easier to follow, but it would also be less consistent with the property that governs telemetry (and the default behavior is to have monitoring enabled, hence the property to disable it).

Reply:
Yep, I always spend a couple of extra brain cells on computing the actual value of telemetry enabled/disabled because of that :D 

OK, we can leave it as is I guess, for the sake of consistency 😢